frame_id: "world_origin"

body:
  frame_id: "fcu"

timer_main:
  rate: 5.0 # [Hz]

# School
bounded_box:
  min_x: -25.0 # [m]
  max_x: 25.0 # [m]
  min_y: -20.0 # [m]
  max_y: 15.0 # [m]
  min_z: 1.0 # [m]
  max_z: 20.0 # [m]
  planner_range: 3 # [m]

rrt:
  N_max: 50
  N_termination: 300
  N_yaw_samples: 10
  radius: 2.0 # [m]
  step_size: 2.0 # [m]
  tolerance: 1.5 # [m]

camera:
  frame_id: "rgbd_front_pitched/aligned_depth_to_color"
  h_fov: 1.51844 # [rad]
  width: 1080
  height: 720
  min_distance: 0.2 # [m]
  max_distance: 5 # [m]

path:
  uav_radius: 1.5 # [m]
  lambda: 0.25 # (tunning factor)
  lambda2: 0.5 # (tunning factor)
  max_acceleration_iterations: 3

reference:
  latitude: 47.397743 #38.736667 # [deg]
  longitude: 8.5455941 #-9.138611 # [deg]
  altitude: 47.29491589289246 #0 # [m]
